lib/screens/recruit/search/project_search_text_field.dart

```diff
@@ -0,0 +1,114 @@
+              maxLines: 1,
+              autofocus: false,
+              controller: controller,
+              keyboardType: TextInputType.multiline,
+              textInputAction: TextInputAction.search,
+              cursorColor: GuamColorFamily.purpleCore,
+              style: TextStyle(
+                fontSize: 14,
+                fontFamily: GuamFontFamily.SpoqaHanSansNeoRegular,
+                color: GuamColorFamily.grayscaleGray1,
+                height: 22/14,
+              ),
+              decoration: InputDecoration(
+                hintText: "검색어를 입력해주세요.",
+                prefixIcon: Icon(
+                  Icons.search_outlined,
+                  color: GuamColorFamily.purpleCore,
+                  size: 20,
+                ),
+                suffixIcon: !isTextEmpty
+                    ? IconButton(
+                  padding: EdgeInsets.zero,
+                  constraints: BoxConstraints(),
+                  onPressed: () {
+                    controller.clear();
+                    isTextEmpty = true;
+                  },
+                  icon: SvgPicture.asset(
+                    'assets/icons/cancel_filled_x_transparent.svg',
+                    color: GuamColorFamily.grayscaleGray6,
+                    width: 18,
+                    height: 18,
+                  ),
+                )
+                    : null,
+                contentPadding: EdgeInsets.symmetric(vertical: 8, horizontal: 12),
+                border: OutlineInputBorder(
+                  borderRadius: BorderRadius.circular(10),
+                  borderSide: BorderSide(color: GuamColorFamily.purpleLight2),
+                ),
+                enabledBorder: OutlineInputBorder(
+                  borderRadius: BorderRadius.circular(10),
+                  borderSide: BorderSide(color: GuamColorFamily.purpleLight2),
+                ),
+                focusedBorder: OutlineInputBorder(
+                  borderRadius: BorderRadius.circular(10),
+                  borderSide: BorderSide(color: GuamColorFamily.purpleCore),
+                ),
+              ),
+              onChanged: (e) => isTextEmpty = controller.text.trim() == '',
+              onSubmitted: (word) {
+                if (!isTextEmpty) {
+                  recruitProvider.searchProjects(keyword: word, skill: widget.filter['skill'], due: widget.filter['due'], position: widget.filter['position']);
+                  widget.cancelSearch(false);
+                  FocusScope.of(context).unfocus();
+                }
+              },
+            ),
+          ),
+          Padding(padding: EdgeInsets.only(right: 4)),
+          if (!isTextEmpty || recruitProvider.searchedProjects.isNotEmpty)
+            CommonTextButton(
+                text: '취소',
+                fontSize: 14,
+                textColor: GuamColorFamily.purpleCore,
+                onPressed: () {
+                  Navigator.pop(context);
+                }
```

Comment:
@kwish2 소원님 혹시 '취소' 누르고 다시 들어갔을 때 일부러 기존 검색어를 textField에 남기신게 아니라면 아래처럼 clear하면 좋을 것 같습니다! 이거 확인하는 것 빼곤 approve입니당!

```suggestion
                onPressed: () {
                  controller.clear();
                  isTextEmpty = true;
                  Navigator.pop(context);
                }
```